packages/forms-web-app/src/routes/config.js

```diff
@@ -178,7 +178,18 @@ const routesConfig = {
 				id: 'examination-select-file',
 				name: 'Select a file',
 				route: '/select-a-file',
-				view: 'pages/examination/select-file'
+				view: 'pages/examination/select-file',
+				onError: {
+					message: {
+						isEmpty: (fileToUploadName = '') => `${fileToUploadName} is empty`,
```

Comment:
are these necessary in config ? 

They can be in the error functions hard coded ?